src/app/containers/RadioPageBlocks/Blocks/OnDemandHeading/index.jsx

```diff
@@ -37,7 +37,12 @@ const Datestamp = styled.span`
   margin: 0;
 `;
 
-const HeadingContainer = ({ idAttr, brandTitle, releaseDateTimeStamp }) => {
+const HeadingContainer = ({
+  idAttr,
+  brandTitle,
+  releaseDateTimeStamp,
+  ariaHidden,
```

Comment:
Whata aria hidden for?

Reply:
On the radio page, we need this component to be an `<h1>` and read out by screen readers
On the TV page, we need this component to be a `<strong>` and read out by screen readers

This boolean prop is how we toggle that